tox.ini

```diff
@@ -1,13 +1,16 @@
 [tox]
 envlist = flake8, {py27,pypy,py33,py34}-{twtrunk,twcurrent}
 
+[flake8]
+ignore = E501
+exclude = crossbar/worker/test/examples/syntaxerror.py
 
 [testenv:flake8]
 skip_install = True
 deps =
    flake8
 commands =
-   flake8 --ignore=E501 crossbar
+   flake8 crossbar
```

Comment:
Flake8 looks at the [flake8] section in tox.ini automatically -- so we don't need to define it again. This means we can run flake8 manually and it'll pick up the settings.